app/views/topical_events/show.html.erb

```diff
@@ -58,7 +58,7 @@
 
     <% if @topical_event.image_url %>
       <div class="topical-events__image">
-      <%= image_tag(@topical_event.image_url, alt: @topical_event.image_alt_text, srcset: { @topical_event.image_srcset[2][:url] => "2x", @topical_event.image_srcset[0][:url] => "3x",}) %>
+      <%= image_tag(@topical_event.image_url, alt: @topical_event.image_alt_text, srcset: @topical_event.image_srcset) %>
```

Comment:
What happens if `image_srcset` is empty?

Reply:
browsers ignore the empty srcset attribute and show the image defined in the src-attribute

Reply:
I made it so that srcset is not rendered if there are no image variants to avoid any possible issues with it